Package.swift

```diff
@@ -56,6 +56,11 @@ let package = Package(
       targets: ["JavaKit"]
     ),
 
+    .library(
+      name: "JavaRuntime",
+      targets: [ "JavaRuntime"]
```

Comment:
```suggestion
      targets: ["JavaRuntime"]
```

Need to enable the Swift formatting enforcement soon 🤔

Reply:
Eh, sure